ioservice/ut/cob_foms.c

```diff
@@ -922,7 +926,7 @@ static void cc_cob_create_test()
 	/*
 	 * Test-case 1 - Verify COB creation
 	 */
-	cob_verify(fom, true);
+	cob_verify(fom, true, false);
```

Comment:
Please add comment here to explain:
the `test_cob` is not released, because it is used later in this test to do cob_delete.

Reply:
Added.

Reply:
OK